From a03058ed605345cce9d7f0cd98f7c3696c3d7c51 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Tue, 7 Apr 2020 13:44:37 +0800
Subject: [PATCH] fix installation of cmake modules

Make sure that cmake files are installed correctly and don't install
license files.

Upstream-Status: Pending

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 CMakeLists.txt | 14 ++++----------
 1 file changed, 4 insertions(+), 10 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 604daa0e45..b60a013b86 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -667,12 +667,12 @@ if (NGRAPH_EXPORT_TARGETS_ENABLE)
     install(EXPORT ngraphTargets
         FILE ngraphTargets.cmake
         NAMESPACE ngraph::
-        DESTINATION cmake
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/ngraph
         COMPONENT ngraph)
 
     configure_package_config_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/share/ngraphConfig.cmake.in
         ${CMAKE_CURRENT_BINARY_DIR}/ngraphConfig.cmake
-        INSTALL_DESTINATION cmake)
+        INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/ngraph)
 
     write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/ngraphConfigVersion.cmake
         VERSION ${NGRAPH_VERSION}
@@ -680,14 +680,8 @@ if (NGRAPH_EXPORT_TARGETS_ENABLE)
 
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ngraphConfig.cmake
         ${CMAKE_CURRENT_BINARY_DIR}/ngraphConfigVersion.cmake
-        DESTINATION cmake
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/ngraph
         COMPONENT ngraph)
 endif()
 
-install(DIRECTORY
-    ${CMAKE_CURRENT_SOURCE_DIR}/licenses
-    DESTINATION "."
-    COMPONENT ngraph
-)
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE DESTINATION "." COMPONENT ngraph)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/VERSION DESTINATION "." COMPONENT ngraph)
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/ngraph COMPONENT ngraph)
-- 
2.25.2

